Removes python 3.9 (#57)
opendp/tumult-analytics: CHANGELOG.rst

Changed
~~~~~~~

> - Dropped support for Python 3.9 as is reached its end-of-life support.

Reviewer: ```suggestion
- Dropped support for Python 3.9, as it has reached end-of-life.
```
